test/request-signal.js

```diff
@@ -74,3 +78,20 @@ test('post abort signal w/ reason', async (t) => {
   })
   await t.completed
 })
+
+test('post abort signal after request completed', async (t) => {
+  t = tspl(t, { plan: 1 })
+
+  const server = createServer((req, res) => {
+    res.end('asd')
+  })
+  after(() => server.close())
+
+  server.listen(0, async () => {
+    const ac = new AbortController()
+    const ures = await request(`http://0.0.0.0:${server.address().port}`, { signal: ac.signal })
+    ac.abort()
+    t.equal(await ures.body.text(), 'asd')
```

Comment:
ℹ️ in this case the abort signal is ignored since the request is completed

Reply:
Since these are modeled after the fetch body mixins, I'm wondering if they should throw. In the fetch spec, if a request was aborted and you attempt to read the body, it'll throw an error. @ronag wdyt?